signals.go

```diff
@@ -48,10 +48,6 @@ func GetStopSignal(ctx context.Context, container Container, defaultSignal sysca
 
 // GetOCIStopSignal retrieves the stop signal specified in the OCI image config
 func GetOCIStopSignal(ctx context.Context, image Image, defaultSignal string) (string, error) {
-	_, err := ParseSignal(defaultSignal)
```

Comment:
I'm paging back in this work but why was this removed? If it actually serves no purpose/can be removed with this work, can we remove the `defaultSignal` parameter since it's no longer used anywhere else in this call?

Reply:
I decided to update the function a bit to actually parse the signal for either the running platform or the platform of the image, so as to not break expected behavior with this (though it's only used in ctr so not a massive problem).